source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,90 +1,83 @@
+Then the [`govuk_crawler_worker`](https://github.com/alphagov/govuk_crawler_worker) on the mirrorer machine:
 
-Every hour, the static copy of the site is copied from the Mirrorer machine to the primary AWS S3 bucket
-`govuk-<environment>-mirror`. This primary bucket is automatically replicated to another S3 bucket
-(named `govuk-<environment>-mirror-replica`) in another region by AWS.
+- consumes the fetched GOV.UK URLs
+- retrieves the GOV.UK HTML files returned by these URLs
+- saves these HTML files to a `/tmp` folder on the mirrorer machine
+- adds any new URLs found on those pages to the back of the message queue
 
-In addition, the primary AWS S3 bucket `govuk-<environment>-mirror` is synced to the Google GCS bucket
-of the same name daily at 12:00.
+Every hour, the [`govuk_sync_mirror`](https://github.com/alphagov/govuk-puppet/blob/86d1480c6e081313c415246063d5931af24473da/modules/govuk_crawler/manifests/init.pp#L109) script runs on the mirrorer machine. This script copies the GOV.UK HTML files from the mirrorer machine to the main `govuk-<environment>-mirror` AWS S3 bucket. AWS then copies this main bucket to the replica `govuk-<environment>-mirror-replica` S3 bucket in another region.
 
-The crawler is entirely independent of the mirrors. Stopping the crawler means no new updates are made
-to the mirrors, but it will not stop the mirrors from working.
+Finally, every day at 6:00pm, a job is run in Google Cloud Storage (GCS) which syncs the primary AWS S3 bucket `govuk-<environment>-mirror` to the GCS `govuk-<environment>-mirror` bucket. For more information on GCS, see the [Google Cloud Platform documentation](https://docs.publishing.service.gov.uk/manual/google-cloud-platform-gcp.html).
 
-To inspect the contents of the mirror:
+The `govuk_seed_crawler` and `govuk_crawler_worker` scripts are independent of the mirrors. Stopping these scripts stops the mirror updates, but does not stop the mirrors from working.
 
-```bash
+Run the following to inspect the contents of the GOV.UK mirrorer machine:
+
+```
 gds govuk connect -e production ssh mirrorer
 cd /mnt/crawler_worker/www.gov.uk
 ```
 
+## “Mirror GOV.UK content to S3” alert
+
+If the govuk_sync_mirror cronjob has not succeeded for 24 hours, it triggers the “Mirror GOV.UK content to S3” alert. See the [Mirror GOV.UK content to S3 alert documentation](/manual/alerts/mirror-sync.html) for more information.
+
 ## Forcing failover to the static mirrors
 
-Because the CDN will retry every request against the mirrors automatically if origin is unavailable,
-[stopping Nginx on the cache machines with Fabric][fab-fail] will result in falling back to mirrors:
+Because Fastly will retry every request against the mirrors automatically if Origin is unavailable, [stopping Nginx on the cache machines with Fabric][fab-fail] will result in user requests going to the mirror. Run the following command to stop Nginx on the cache machines:
```

Comment:
We're no longer recommending Fabric as a tool, so this section will eventually need a small rewrite. That can happen after this PR though. @huwd is gradually working through the docs to remove references to Fabric.

Reply:
Sounds good, will leave for now